tools/protein_properties/protein_properties.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
+
+import sys
+from Bio import SeqIO
+from Bio.SeqUtils.ProtParam import ProteinAnalysis
+
+with open(sys.argv[2], 'w+') as result:
```

Comment:
Just a thought, I've been writing a lot of tools lately and I've been printing, or writing to `sys.stdout` in lieu of opening extra files. Having more pipeable tools is nice.